modules/platform/nuxeo-platform-document-routing/nuxeo-routing-core/src/test/java/org/nuxeo/ecm/platform/routing/test/TestDocumentRoutingService.java

```diff
@@ -384,6 +385,31 @@ public void testOrphanTasksDeletion() {
         assertFalse(session.exists(task.getRef()));
     }
 
+    @Test
+    public void testOrphanDocumentRoutesDeletion() {
+        DocumentModel file1 = session.createDocumentModel("/", "File1", "File");
+        file1 = session.createDocument(file1);
+        file1 = session.saveDocument(file1);
```

Comment:
`saveDocument` is useless if `createDocument` was called just before and no property was updated